### PR TITLE
[plugins/js-eval] add deno environment

### PR DESCRIPTION
**Pull request by stagas**
This PR adds support for [deno](https://deno.com/) [eval](https://docs.deno.com/runtime/manual/tools/eval) using the `d>` prefix. 

Deno is a modern JavaScript runtime with builtin support for TypeScript, isomorphism with many standard Web APIs and [many more](https://deno.land/api?unstable=true).

Examples to try:

`d> Deno.version`
`d> let x: string = 'hello world'; x`

### CODE REVIEW

File: src/plugins/js-eval/Dockerfile [hunk 1]
```diff
@@ -15,12 +15,17 @@ RUN node --version
 RUN npm i --prefix /run airbnb-js-shims@latest string.prototype.at@latest array.prototype.at@latest @bloomberg/record-tuple-polyfill@latest nearley@latest array.prototype.group@latest array.prototype.grouptomap@latest array.prototype.flat@latest array.prototype.flatmap@latest array.prototype.with@latest array.prototype.tosorted@latest array.prototype.toreversed@latest array.prototype.tospliced@latest set.prototype.union set.prototype.intersection set.prototype.difference set.prototype.symmetricdifference set.prototype.issubsetof set.prototype.issupersetof set.prototype.isdisjointfrom es-set es-map suppressed-error es-iterator-helpers arraybuffer.prototype.detached disposablestack
 RUN curl https://engine262.js.org/engine262/engine262.js -o /run/node_modules/engine262.js
 
-FROM debian:10-slim
-
 RUN useradd node && mkdir -p /home/node && chown -R node:node /home/node
 WORKDIR /home/node
-COPY --from=0 /run /run
-COPY --from=0 /usr/local/bin/node /usr/local/bin/node
```
Comment (stagas) [inline src/plugins/js-eval/Dockerfile:23]: These `COPY --from=0 ..` commands were failing locally. I don't think there is a reason for them to be there as well?

File: src/plugins/js-eval/Dockerfile [hunk 2]
```diff
@@ -15,12 +15,17 @@ RUN node --version
 RUN npm i --prefix /run airbnb-js-shims@latest string.prototype.at@latest array.prototype.at@latest @bloomberg/record-tuple-polyfill@latest nearley@latest array.prototype.group@latest array.prototype.grouptomap@latest array.prototype.flat@latest array.prototype.flatmap@latest array.prototype.with@latest array.prototype.tosorted@latest array.prototype.toreversed@latest array.prototype.tospliced@latest set.prototype.union set.prototype.intersection set.prototype.difference set.prototype.symmetricdifference set.prototype.issubsetof set.prototype.issupersetof set.prototype.isdisjointfrom es-set es-map suppressed-error es-iterator-helpers arraybuffer.prototype.detached disposablestack
 RUN curl https://engine262.js.org/engine262/engine262.js -o /run/node_modules/engine262.js
 
-FROM debian:10-slim
```
Comment (stagas) [inline src/plugins/js-eval/Dockerfile:18]: This is done earlier and was causing `curl` to not be available for the deno download later.

File: src/plugins/js-eval/__tests__/jsEvalPlugin-test.js [hunk 3]
```diff
@@ -130,7 +130,7 @@ describe('jsEvalPlugin', () => {
     });
   });
 
-  describe('engine262', () => {
+  xdescribe('engine262', () => {
```
Comment (stagas) [inline src/plugins/js-eval/__tests__/jsEvalPlugin-test.js:133]: These tests will need to be updated for latest engine262 and reenabled. They were failing both locally and on GH. They were disabled in [a separate commit](https://github.com/ljharb/jellobot/pull/15/commits/69a7b90501db5309fd882fadf7f7748ef7380301)

File: src/plugins/js-eval/Dockerfile (hunk 2 above)
Comment (ljharb) [reply src/plugins/js-eval/Dockerfile:18]: yeah it's weird the line was duplicated

File: src/plugins/js-eval/Dockerfile (hunk 1 above)
Comment (ljharb) [reply src/plugins/js-eval/Dockerfile:23]: it seems like it's a permissions resetting thing. we can try it, but this might make deployment fail.

**Review by ljharb — APPROVED** [review]

**Pull request merged**